charts/auth-service/templates/ingressroute.yaml

```diff
@@ -14,8 +14,9 @@ spec:
         - name: {{ include ".helm.fullname" . }}
           kind: Service
           port: {{ .Values.image.containerPort }}
-      {{ if (not (hasKey .Values.image "noStripPath")) }}
       middlewares:
+      - name: "{{ include ".helm.fullname" . }}-headers"
+      {{ if (not (hasKey .Values.image "noStripPath")) }}
```

Comment:
I know that I put this whole noStripPath in but honestly I hate it and have been meaning to take it out... Do we need to do any strippath middleware for auth service? If not, lets ditch it